canonical-web-and-design/maas-ui
# Add FQDN column content

## Done
- Display FQDN, MAC address and ip addresses in the machine list.

## QA
- Visit /r/machines on a maas with multiple machines (or use sample data).
- You should see the FQDN and ip addresses for your machines in the first column.
- Inspect the react component for that cell on one of the machines and turn on `showMAC`. You should . now see the mac address for that machine.

## Fixes
Fixes: https://github.com/canonical-web-and-design/maas-squad/issues/1754.
